libcst/metadata/tests/test_scope_provider.py

```diff
@@ -596,3 +598,81 @@ def test_with_asname(self) -> None:
                 ensure_type(m.body[0], cst.With).items[0].asname, cst.AsName
             ).name,
         )
+
+    def test_get_fully_qualified_names_for(self) -> None:
```

Comment:
Can we add a simple test for nested classes and nested functions? I don't see that tested and it looks like from your code that you mean to handle it.

Reply:
Added test_get_qualified_names_for_nested_cases.